cmd/kubeadm/app/apis/kubeadm/v1beta3/doc.go

```diff
@@ -23,7 +23,8 @@ limitations under the License.
 // This version improves on the v1beta2 format by fixing some minor issues and adding a few new fields.
 //
 // A list of changes since v1beta2:
-//	- TODO
+// - The deprecated "ClusterConfiguration.useHyperKubeImage" field has been removed.
+// Kubeadm no longer supports the hyperkube image.
```

Comment:
LGTM.

It will have a merge conflict with #101537 merged. 
So I will add /lgtm later.